### PR TITLE
added AdditionalInfo component to Fract

### PR DESCRIPTION
**Pull request by lbai001**
additionalinfo component added

### CODE REVIEW

File: src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  // for the sake of more tests could be written
+  // declared beforeEach and assigned to wrapper
+  beforeEach(() => {
+    wrapper = mount(<AdditionalInfo triggerText="test"/>);
+  });
+
+  it('should render', () => {
+    expect(wrapper.text()).to.contain('test');
+  });
+  it('should pass aXe check', () => {
```
Comment (jbalboni) [inline src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx]: Can we also write tests that run axe on the open state?

File: src/components/AdditionalInfo/AdditionalInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <ExpandingGroup open expandedContentId={this.expandedContentId}>
+        {trigger}
+        <div className="additional-info-content">{children}</div>
+      </ExpandingGroup>
+    );
+  }
+}
+
+AdditionalInfo.propTypes = {
+  triggerText: PropTypes.string
```
Comment (jbalboni) [inline src/components/AdditionalInfo/AdditionalInfo.jsx]: Can you add some documentation to this prop type? You can look at the other components in here for examples. Also, we probably want to make this prop required (probably should have been done in vets-website).

File: src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx (hunk 1 above)
Comment (lbai001) [reply src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx]: okay

File: src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    return axeCheck(<AdditionalInfo triggerText="test"/>);
+  });
+  it('renders both children when open is true', () => {
+    const first = wrapper.find('ExpandingGroup').props();
+    expect(first.open).to.eql(true);
```
Comment (jbalboni) [inline src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx]: You could use `.to.be.true` here.

**Pull request merged**